model/stats/query.go

```diff
@@ -97,17 +97,17 @@ func (s *StartAt) validateCommon(groupBy GroupBy) error {
 	switch groupBy {
 	case GroupByDistro:
 		if len(s.Distro) == 0 {
-			catcher.New("missing distro 'start at' date")
+			catcher.New("missing distro pagination value")
 		}
 		fallthrough
 	case GroupByVariant:
 		if len(s.BuildVariant) == 0 {
-			catcher.New("missing build variant 'start at' date")
+			catcher.New("missing build variant pagination value")
 		}
 		fallthrough
 	case GroupByTask:
 		if len(s.Task) == 0 {
-			catcher.New("missing task 'start at' date")
+			catcher.New("missing task pagination value")
 		}
 	}
 	return catcher.Resolve()
```

Comment:
I couldn't put a comment on the line, but below on L121 change it to `missing test pagination value`.